rc/pdfpcrc

```diff
@@ -1,129 +1,123 @@
 #### Keybindings
 
-# Forward movement
-bind Next      next
-bind Return    next
-bind Right     next
-bind space     next
-bind l         lastOverlay
-bind Down      nextOverlay
-bind S+Next    next10
-bind S+Return  next10
-bind S+Right   next10
-bind S+space   next10
-bind S+Down    next10
-bind k         nextUnseen
-
-# Back movement
-bind Left      prev
-bind Prior     prev
-bind L         firstOverlay
-bind Up        prevOverlay
-bind S+Left    prev10
-bind S+Prior   prev10
-bind S+Up      prev10
-bind j         prevSeen
+# Standard forward movement
+bind Next               next
+bind Return             next
+bind Right              next
+bind space              next
+bind Down               next
+
+# Standard backward movement
+bind Prior              prev
+bind Left               prev
+bind Up                 prev
+bind BackSpace          prev
+
+# Overlay paging
+bind S+Next             nextOverlay
+bind S+Prior            prevOverlay
+
+# Intra-slide navigation
+bind S+Up               lastOverlay
+bind S+Down             firstOverlay
+
+# Fast forward/backward (10 pages)
+bind S+Right            next10
+bind S+Left             prev10
 
 # "Jumps"
-bind Home      gotoFirst
-bind End       gotoLast
-bind g         goto
-bind Tab       overview
-bind BackSpace histBack
+bind Home               gotoFirst
+bind End                gotoLast
+bind g                  goto
+
+# Bookmarking [the current action names are misleading]
+bind m                  lastSlide
+bind S+m                jumpLastSlide
+
+# History? [these 3 are seriously broken]
```

Comment:
Remove it?

Reply:
The `nextUnseen`/`prevSeen` pair come from a specific request - #242, but either they're broken, or I don't understand the logic.

About `histBack` - it works correctly when you press it one time (returns from a `goto` jump), but then it works randomly. I haven't looked inside yet. The history functionality, if implemented correctly (including forward traversal - `S+Space`?), might be a nice addition. At least in ordinary pdf viewers I use it quite frequently.